File: haul3/data/platforms/dos/libs/hio.pas
{ HAUL IO }
Unit hio;
{$N+}

Interface

Procedure put(data:String);
Procedure put_(data:String);
Procedure shout(data:String);
Function fetch:String;

Function int_str(i:Integer):String;
Function float_str(f:Single):String;

Implementation

Procedure put(data:String);
Begin
	WriteLn(data);
End;
Procedure put_(data:String);
Begin
	Write(data);
End;
Procedure shout(data:String);
Begin
	WriteLn('!! ', data);
End;

Function fetch:String;
var
	s: String;
begin
	readln(s);
	fetch := s;
end;


Function int_str(i:Integer):String;
Var
	s:String;
begin
	Str(i, s);
	int_str := s;
End;

Function float_str(f:Single):String;
var
	s: String;
begin
	Str(f, s);
	float_str := s;
End;

End.

File: haul3/data/langs/py/libs/hio.py
# HAUL IO

#import sys

def put(data):
	#sys.stdout.write(data)
	print(data)

def put_(data):
	#sys.stdout.write(data)
	print(data),

def shout(data):
	print('!! ' + str(data))

def fetch():
	# PY3
	#return input()
	
	data = raw_input()
	return data

def int_str(i):
	return str(i)
def float_str(i):
	return str(i)